packages/moonstone/SlotItem/SlotItem.js

```diff
@@ -63,6 +63,7 @@ const SlotItemBase = kind({
 	render: (props) => {
 		return (
 			<UiSlotItemBase
+				data-webos-voice-intent="Select"
```

Comment:
This isn't needed. `SlotItem` uses `Item` which already has this prop applied.

Reply:
I will remove it.